google/cloud/aiplatform/models.py

```diff
@@ -1326,15 +1364,508 @@ def explain(
+        """
+        try:
+            import urllib3
+        except ImportError:
+            raise ImportError(
+                "Cannot import the urllib3 HTTP client. Please install google-cloud-aiplatform[private_endpoints]."
+            )
+
+        try:
+            response = self._http_client.request(
+                method=method, url=url, body=body, headers=headers
+            )
+
+            if response.status < 300:
+                return response
+            else:
+                raise RuntimeError(
+                    f"{response.status} - Failed to make request, see response: "
+                    + response.data.decode("utf-8")
+                )
+
+        except urllib3.exceptions.MaxRetryError as exc:
+            raise RuntimeError(
+                f"Failed to make a {method} request to this URI, make sure: "
+                " this call is being made inside the network this private Endpoint is peered to "
+                f"({self._gca_resource.network}), calling health_check() returns True, "
+                f"and that {url} is a valid URL."
+            ) from exc
+
+    def predict(self, instances: List, parameters: Optional[Dict] = None) -> Prediction:
+        """Make a prediction against this private Endpoint using a HTTP request.
+        This method must be called within the network the private Endpoint is peered to.
+        The predict() call will fail otherwise. To check, use `PrivateEndpoint.network`.
+
+        Example usage:
+            response = my_private_endpoint.predict(instances=[...])
+            my_predictions = response.predictions
+
+        Args:
+            instances (List):
+                Required. The instances that are the input to the
+                prediction call. A DeployedModel may have an upper limit
+                on the number of instances it supports per request, and
+                when it is exceeded the prediction call errors in case
+                of AutoML Models, or, in case of customer created
+                Models, the behaviour is as documented by that Model.
+                The schema of any single instance may be specified via
+                Endpoint's DeployedModels'
+                [Model's][google.cloud.aiplatform.v1beta1.DeployedModel.model]
+                [PredictSchemata's][google.cloud.aiplatform.v1beta1.Model.predict_schemata]
+                ``instance_schema_uri``.
+            parameters (Dict):
+                The parameters that govern the prediction. The schema of
+                the parameters may be specified via Endpoint's
+                DeployedModels' [Model's
+                ][google.cloud.aiplatform.v1beta1.DeployedModel.model]
+                [PredictSchemata's][google.cloud.aiplatform.v1beta1.Model.predict_schemata]
+                ``parameters_schema_uri``.
+
+        Returns:
+            prediction: Prediction with returned predictions and Model Id.
```

Comment:
Seems like we have no consistent way to denote return parameters.

Prefer this format as it includes the type. Example:

```
        Returns:
            prediction_client (prediction_service_client.PredictionServiceClient):
                Initialized prediction client with optional overrides.
```